.buildkite/scripts/steps/on_merge_build_and_metrics.sh

```diff
@@ -2,6 +2,9 @@
 
 set -euo pipefail
 
+# Write Bazel cache for Linux
+.buildkite/scripts/common/setup_bazel.sh
```

Comment:
any plans to enable this for macOS soon too?

Reply:
Yeah, no timeline yet, though